src/ontology/uberon-edit.obo

```diff
@@ -223695,17 +223690,14 @@ is_metadata_tag: true
 
 [Typedef]
 id: attaches_to
-name: attaches_to
+name: attached_to
```

Comment:
Why the label change?

Reply:
Because the source is this one that I changed.

Reply:
This one is being used in many terms. It's a symmetric property.